### ISSUE TITLE
integrate EDAM topics into Dataset Type

### ISSUE THREAD

**Issue by only1chunts** [gigascience/gigadb-website#288]
The dataset type list is current a short controlled vocabulary made up by us based on general topics of the datasets we host. To enable a more expansive but still controlled set of terms and to allow us to integrate with others, we should use the Topic section of the EDAM ontology.
http://edamontology.org/topic_0003

or to view it in OLS:
https://www.ebi.ac.uk/ols/ontologies/edam/terms?iri=http%3A%2F%2Fedamontology.org%2Ftopic_0003

**Comment by only1chunts** [comment]
An alternative to EDAM might be The Subject Resource Application Ontology (SRAO) https://github.com/FAIRsharing/subject-ontology.  This is a combination of EDAM with intergration/mapping to other ontologies.
And  the Domain Resource Application Ontology (DRAO) https://github.com/FAIRsharing/domain-ontology , which is more granular than SRAO.

**Comment by only1chunts** [comment]
have created a slim of the SRAO and grouped into 13 primary classifications, see spreadsheet
https://drive.google.com/file/d/1shH-QK-UT9HD88ysWFHVq5xUenxAmcbg/view?usp=sharing
This is likely to be the starting set of terms for the primary and secondary claffications in RV submission tool. We will need to decide which terms are included in GigaDB initially, do the mappings for any deprecated terms and potentially update all datasets with appropriate new terms?

**Comment by ScottBGI** [comment]
Not sure if this is helpful for how we design these, and it could be a useful mapping exercise project for an intern, but I've just seen Dryad use domains drawn from the OECD Fields of Science and Technology classification: https://www.oecd.org/science/inno/38235147.pdf

**Comment by only1chunts** [comment]
I dont believe using the OECD classifications is the way for us to go as its a very high-level classification, but I have opened a [ticket in the SRAO](https://github.com/FAIRsharing/subject-ontology/issues/73) project to ask them to look into mapping their terms to it!
FYI- the OECD have a newer version (than the one Scott pointed to) of the controlled vocabulary of terms relating to the fields of R&D (FORD) classifications in the main Frascati manual (see table 2.2 in [this document](https://read.oecd-ilibrary.org/science-and-technology/frascati-manual-2015_9789264239012-en#page61))
